### DISCUSSION TITLE
Storybook 7 LTS?

### DISCUSSION THREAD

**Discussion by mnicholstamu** [storybookjs/storybook#26455] [General] [answered]
I was just wondering how long Storybook 7 will be around before we will have to move to Storybook 8? I have been experimenting with SB8 and I am loving it, but our code base will need to be updated and that will take some time, we are a bit of a small shop right now. Thank you for making an awesome product!

**Comment by valentinpalkovic** [comment, marked as answer]
Hi @mnicholstamu 

I'm glad to hear that you're enjoying Storybook 8! 

Please keep in mind that we typically provide only bug fixes and feature implementations for the latest major version, Storybook 8. The previous version of Storybook, Storybook 7, will only receive critical bug fixes or security vulnerability patches. Still, it's best to prioritize transitioning to the latest version to leverage all the newest features and enhancements. 

I understand that updating your code base can take some time, especially for smaller teams, so feel free to reach out if you need any assistance or guidance during the transition process. 

Thank you for your kind words about our product - we're thrilled to have you on board!
